.gitignore

```diff
@@ -30,4 +30,5 @@ python/package-lock.json
 python/package.json
 python/keys.json
 python/LICENSE.txt
-.env
\ No newline at end of file
+.env
+js/
```

Comment:
We'd rather keep the js files updated in the repo, Travis pushes them back for py/php, so it could do the same with js.